.eslintrc.js

```diff
@@ -69,6 +69,13 @@ module.exports = {
           }
         ]
       }
+    },
+    {
+      files: ['*.ts'],
+      rules: {
+        'no-unused-vars': 'off',
+        '@typescript-eslint/no-unused-vars': ['error']
+      }
```

Comment:
could the services/web-app/lib/file-cache.js sonarcloud error be caused by this change?

Reply:
Nah, that was just me being dumb and editing a change on the github UI rather than in my IDE. I commented out the only use of a var, but didn't comment out the var definition itself.

This change is needed because the baseline eslint rule for unused vars doesn't properly handle typescript types and a couple of other weird edge cases.